### ISSUE TITLE
Some questions about training

### ISSUE THREAD

**Issue by Sakuralala** [timctho/convolutional-pose-machines-tensorflow#18]
Hey I used a different network architecture(resnet-50) but the same method as you do,but when I run training,I had 2 questions:
1、During training,at first the loss went down quickly, from 10e7 to 10e4,but after that,the loss fall into shock and can't go down to smaller easily.My question is, when you train the net,how does the loss change?Does my loss change normally?And What the value of loss after your training finished?(By the way,the learning rate and some other parameters are copyed from your config.py)
2、 I'm not sure how to make the heatmap for the background class,in your code,I see you just make a nxn matrix whichs' value are all 1,and then you reduce all other heatmapss' value ele-wise.And is the gaussian_varianve a important parameter?Because when I changed it bigger,the net seems more difficult to converge.

**Comment by timctho** [comment]
1. It depends on your output heat map size, because it uses L2 loss. It my case, if output size is 32x32, final loss is around 50 each stage.
2. Yes I subtract all the finger joint gaussians, the rest is background. Variance does affect a lot, it is a little bit tricky to tune, too large means you're giving network too much tolerance, too little is tend to overfit due to labeling errors.
